### ISSUE TITLE
The convergence issue during the fine-tuning process

### ISSUE THREAD

**Issue by Wangweizhen-S** [yyfz/Pi3#140]
I fine-tuned my dataset using the pi3. I adopted a high-resolution configuration and disabled the confidence head. I noticed that the pose quickly converged, but the points became blurry. However, when using the pre-trained weights of pi3 for prediction, the local point cloud for each frame had a normal shape. Why is that? What are the things to note during the fine-tuning process?

<img width="709" height="674" alt="Image" src="https://github.com/user-attachments/assets/2b2abcad-c611-41a8-b235-effee7b4543e" />（This is the fintuned result.）

<img width="1279" height="904" alt="Image" src="https://github.com/user-attachments/assets/62597c4b-5a12-4c8b-b915-f7ec28237263" />(This is the pretrained model‘s result.)
The pretraining results have a general shape (perhaps the depth is not precise enough), but the pose is incorrect. Now, after fine-tuning, the results have a more accurate pose, but the depth is lost. And the training loss even still lower, and the validation loss is also much lower than the first few training iterations but remains unchanged after few epochs.

**Comment by yyfz** [comment]
Hi, I think this is mainly because the data scale and quality.

**Comment by Wangweizhen-S** [comment]
> Hi, I think this is mainly because the data scale and quality.
I have already finetuned successfully vggt of the same dataset before.

**Comment by Wangweizhen-S** [comment]
> Hi, I think this is mainly because the data scale and quality.I would like to know if it is possible to simply perform the fine-tuning directly in accordance with the full-scale fine-tuning of the second stage?

**Comment by yyfz** [comment]
Can you overfit one sample to check if there is some bugs? I think the results are not normal.

**Comment by Wangweizhen-S** [comment]
<img width="1157" height="830" alt="Image" src="https://github.com/user-attachments/assets/43ede6a1-4993-473a-abb4-bff93134ad27" />The current result of the reasoning shows the general shape, but the details are overly blurred. What could be the reason for this?

**Comment by yyfz** [comment]
Keep training and wait for the learning rate to drop.

**Comment by Wangweizhen-S** [comment]
> Keep training and wait for the learning rate to drop.

Thank you very much for your reply. All the problems have been solved and your response was very helpful. In addition to this, I would like to ask about the fine-tuning of the object category dataset. How is the background outside the objects in the input photos handled? Use a black background directly?

**Comment by yyfz** [comment]
Glad the previous info helped! I think using a black or white background is both no problem.
